whylog/assistant/regex_assistant/regex_object.py

```diff
@@ -8,69 +10,80 @@ class RegexObject(object):
     Supports interactive updates like adding/removing groups to regex, replacing regex
     Keeps own data integrity - especially consistency between regex and groups (group_spans).
     Verifies updates.
-    Not responsible for guessing where are groups in regex
+
+    :param line_text: line text (raw string)
+    :param param_groups: represents params from text (catched by regex groups)
+    :param regex: regex matching to line_text
+    :param guessed_pattern_objects: keeps guessed regexes that match to line_text
```

Comment:
Again. Why do you use the _objects suffix? guessed_patterns doesn't say the same?